### ISSUE TITLE
Better accessibility

### ISSUE THREAD

**Issue by helgefredheim** [prakhar1989/react-tags#530]
Great component, however it is not accessible, thus we cannot use it. Would love to contribute, is this of interest?

**Comment by ad1992** [comment]
@helgefredheim yes we want to make it accessible Would be great if you can contribute !!
